src/map/lua/luautils.cpp

```diff
@@ -1,4 +1,4 @@
-﻿/*
```

Comment:
did the BOM on this file get changed?

Reply:
They tend to flip flop all the time, I tend to ignore it now

Reply:
Honestly not sure what happened there.

Reply:
(I have a tool to detect this in tools/, but we aren't using it)

Reply:
editor config as we have it set currently is _supposed_ to auto re-add the bom for cpp files but hell if I know if it actually is working. Even git itself reads and uses that so what editor was used shouldn't matter either, I think...?

oh well. We'll find out if some random person using an east Asian version of windows or visual studio pops in with the error.